### DISCUSSION TITLE
Suggestion: Double height for infill.

### DISCUSSION THREAD

**Discussion by thefirstnighttowl** [supermerill/SuperSlicer#2120] [Community contribution]
It would be great if we could print external and internal surfaces with 1/2 print layer height for finer detail on visible curves.
What i mean is for the outer perimeter print 2 layers of perimeters at half height, then the infil would be at full height.

**Comment by neophyl** [comment]
Already been suggested several times. Slightly different as they only wanted the outer perimeter at half height and the other perimeters at normal height. 

Also what you describe already exists in both super slicer and Prusa slicer, it’s called combine infill. You can choose to combine multiple layers of infill already, so set your layer height to your half height that you want for your perimeters and then choose to combine infill every 2 layers. Functionally that does exactly what you suggested.
